pkg/core/engine/configuration.go

```diff
@@ -17,8 +18,20 @@ func (e *Engine) LoadConfigurations() error {
 
 	for i := range e.Options.Manifests {
 		if e.Options.Manifests[i].IsZero() {
-			ErrNoManifestDetectedCounter++
-			continue
+
+			// If no manifest file is specified, we try to detect one
+			if _, err := os.Stat("updatecli.yaml"); err == nil {
+				e.Options.Manifests[i].Manifests = append(e.Options.Manifests[i].Manifests, "updatecli.yaml")
```

Comment:
Should this be used? 

https://github.com/updatecli/updatecli/blob/9344acd1230cfa7d58c61f2fa2312f6b56a3425a/pkg/core/config/main.go#L35

Reply:
Thanks a lot for your review, you made a good catch.
The code was there but not reachable anymore.

I have additional code cleanup to do.